lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -5038,10 +5038,11 @@ diagnoseMissingAppendInterpolationMethod(NominalTypeDecl *typeDecl) {
   }
 }
 
-SmallVector<ProtocolConformance *, 2>
+std::vector<ProtocolConformance *>
```

Comment:
Should this use the typedef if you're relying on equivalence?

Reply:
Yeah, it should. Good call.